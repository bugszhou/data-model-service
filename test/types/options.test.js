import opts from '../../src/lib/types/options';

it('测试parse转换的配置字段列表 ', function() {
  expect(opts).toMatchSnapshot();
});

