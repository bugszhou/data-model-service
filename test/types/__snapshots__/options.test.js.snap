// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`测试parse转换的配置字段列表  1`] = `
Object {
  "enter": [Function],
}
`;
